README.md

```diff
@@ -1,88 +1,187 @@
-# project-identity-resolver
-Reference implementation of UNTP link resolver and credential store
+# Storage Service
 
-* Will implement [ISO-18975](https://www.iso.org/standard/85540.html)
-* Will support any identifier type
-* Will add credential store capabilites to host DPPs etc.
+The storage service directory contains an Express REST API
+that provides endpoints to encrypt and store Verifiable Credentials and documents.
 
-Development work is expected to start in July 2024.
+## Overview
 
-## Storage Service
+The service offers the following functionality:
 
-This repository contains an Express REST API that provides an endpoint to encrypt and store Verifiable Credentials.
-
-The credentials endpoint provides the following functionality:
-
--   **Hash Computation**: It computes the hash of a given document, ensuring data integrity.
--   **Encryption**: The document is encrypted, adding a layer of security to publicly accessible documents.
--   **Storage**: The encrypted document is stored using the specified storage adapter.
--   **Data Retrieval**: Upon successful storage, the service returns three key pieces of information:
+-   **Hash Computation**:
+    Computes the SHA-256 hash of a given document to ensure data integrity.
+-   **Encryption**:
+    Encrypts the document using AES-256-GCM for enhanced security.
+-   **Storage**:
+    Stores the encrypted document using the specified storage adapter
+    (local file system or Google Cloud Storage).
+-   **Data Retrieval**:
+    Upon successful storage, the service returns:
     -   The hash of the original document.
-    -   A decryption key, essential for decrypting the encrypted document.
+    -   A decryption key for the encrypted document (if applicable).
     -   The URI of the stored encrypted document.
 
-### Prerequisites
+## Prerequisites
 
--   [npm](https://www.npmjs.com/) (>= 9.8.1)
--   [yarn](https://yarnpkg.com/) (>= 1.22.21)
+-   [Node.js](https://nodejs.org/) (v18.18.0)
+-   [Yarn](https://yarnpkg.com/) (>= 1.22.21)
 
-### Environment variables
+## Environment Variables
 
-An example env file `.env.example` can be found in the root directory. Duplicate and rename the file to `.env` and modify the variables as required.
+An example environment file `.env.example` is provided in the storage service directory.
+Copy and rename it to `.env`,
+then modify the variables as required.
+The default values should be sufficient for local development.
 
-### Usage
+## Usage
 
 ```bash
 # Install dependencies
 yarn install
 
-# Run the app
-yarn start
+# Run the app and watch for changes
+yarn dev
 
 # Build the app
 yarn build
```

Comment:
I believe the `yarn build` step should be placed before the `yarn dev` step, as the `yarn dev` step triggers `watch:server`, which relies on the `dist` folder.

Reply:
@huynguyen-hl, good catch.